library/std/src/Std/DurrHoyerLibrary.qs

```diff
@@ -0,0 +1,226 @@
+open Microsoft.Quantum.Intrinsic;
+open Microsoft.Quantum.Canon;
+open Microsoft.Quantum.Math;
+open Microsoft.Quantum.Measurement;
+open Microsoft.Quantum.Arrays;
+open Microsoft.Quantum.Convert;
+open Microsoft.Quantum.Random;
+open Microsoft.Quantum.Core;
+open Microsoft.Quantum.Diagnostics;
```

Comment:
Could you please do a audit of remaining imports after removing the preludes? On a brief look, Diagnostics and Random seem to be unused.